msvc/libgcmonosgen.targets

```diff
@@ -0,0 +1,161 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="monosgen_sources">
+    <ClInclude Include="$(MonoSourceLocation)\mono\sgen\gc-internal-agnostic.h">
+      <ExcludedFromBuild Condition="'$(MONO_TARGET_GC)'!='sgen'">true</ExcludedFromBuild>
+    </ClInclude>
+    <ClCompile Include="$(MonoSourceLocation)\mono\sgen\sgen-alloc.c">
+		  <ExcludedFromBuild Condition="'$(MONO_TARGET_GC)'!='sgen'">true</ExcludedFromBuild>
+	  </ClCompile>
```

Comment:
tabs/spaces

Reply:
I will make a pass over the files and remove some of the editor artifacts.